ntbs-service/Pages/ContactDetails/CaseManager.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@page "/ContactDetails/CaseManager/{Username}"
```

Comment:
I would suggest `/ContactDetails/{Username}` - all users, not just case managers, have contact pages